include/flatbuffers/flexbuffers.h

```diff
@@ -743,7 +745,7 @@ inline Reference GetRoot(const uint8_t *buffer, size_t size) {
 }
 
 inline Reference GetRoot(const std::vector<uint8_t> &buffer) {
-  return GetRoot(buffer.data(), buffer.size());
+  return GetRoot(flatbuffers::vector_data(&buffer), buffer.size());
```

Comment:
In my last review I asked if this could be called `data` (to match the STL function). It already sits in the `flatbuffer` namespace, so I don't feel this needs `vector_` to be clear.
Also, I think the argument should be by `&` rather than `*`, to match STL, and since most uses are const anyway.